core/container/src/main/java/org/wildfly/swarm/container/runtime/RuntimeServer.java

```diff
@@ -181,13 +181,10 @@ private void configureUserSpaceExtensions() {
     }
 
     public void stop() throws Exception {
-        final CountDownLatch latch = new CountDownLatch(1);
-        this.serviceContainer.addTerminateListener(info -> latch.countDown());
-        this.serviceContainer.shutdown();
-
-        latch.await();
```

Comment:
As we're not using this.serviceContainer as part of the stop() anymore, do we even need to hang onto the object?

Looks like, from a quick glance, that we can remove the instance from the class and use locally on start